daemon/cluster/cluster.go

```diff
@@ -91,16 +92,16 @@ const (
 )
 
 // errNoSwarm is returned on leaving a cluster that was never initialized
-var errNoSwarm = fmt.Errorf("This node is not part of a swarm")
+var errNoSwarm = rerrors.New("This node is not part of a swarm")
```

Comment:
Was discussing this with @vdemeester, and just using `errors.New()` (using the github.com/pkg/errors package) would be ok to do in this file (instead of using the `"errors"` package, and aliasing it as `rerrors`).

Can you change that?